modules/contentbox/migrations/2022_10_13_133204_CONTENTBOX-1296-Unique-Site-Slug-Relocation.cfc

```diff
@@ -0,0 +1,26 @@
+component {
```

Comment:
I don't see that being sustainable, long term, as the version marked may not be the version it actually is released under ( e.g. maybe we need to patch master, but we have already created a migration identified with the patch - which means we have to rename the migrations, which will run them again, etc... )

Since the changelog includes the stories addressed and the story ID is in the migtration, I'm not sure I understand the need for the redundancy.  It doesn't seem agile to me to hard-code version numbers in to the migration files.

Reply:
That said, I have renamed the file to expedite this PR

Reply:
That's a good point.  I guess the OCD in me wants them to be controlled. However, I see your point.ALso, I see the point of this directory becoming HUUGE and unmanageable.  I wonder if we can split it into version buckets.  Even if we have to patch master, it will be released under a semantic version.  So that folder could still stand.  I will test with that approach.